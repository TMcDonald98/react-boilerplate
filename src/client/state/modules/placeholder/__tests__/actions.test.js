import {
  setPlaceholderData
} from '../actions';

describe('placeholder/actions', () => {
  describe('setPlaceholderData', () => {
    it('should get the placeholder data', () => {
      const testData = 'test';
      const result = setPlaceholderData(testData);

      expect(result).toMatchSnapshot('setPlaceholderData');
    });
  });
});
