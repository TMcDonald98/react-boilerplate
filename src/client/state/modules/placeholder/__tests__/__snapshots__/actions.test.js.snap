// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`placeholder/actions setPlaceholderData should get the placeholder data: setPlaceholderData 1`] = `
Object {
  "payload": "test",
  "type": "SET_PLACEHOLDER_DATA",
}
`;
